embulk-core/src/main/java/org/embulk/spi/ReporterCloseable.java

```diff
@@ -0,0 +1,9 @@
+package org.embulk.spi;
+
+public interface ReporterCloseable
```

Comment:
Do we really need this `ReporterCloseable` as an independent interface? What do you think having the methods directly in `AbstractReporter`? (The name might be like `(Abstract)InternalReporter`, `(Abstract)ReporterImpl`, `(Abstract)ReporterInstance` or like them...)